.github/github_workflow_scripts/create_internal_pr.py

```diff
@@ -10,9 +10,8 @@
 from utils import (
     get_env_var,
     timestamped_print,
-    load_json,
     get_doc_reviewer,
-    CONTENT_ROLES_PATH
```

Comment:
Is there any other place this is used at?

Reply:
No:

![image](https://github.com/demisto/content/assets/85439776/337ee441-6ec0-4407-9abf-0599cf732ce8)